### ISSUE TITLE
Add FAQ to answer "minimum requirements for device to run resin" 

### ISSUE THREAD

**Issue by alisondavis17** [resin-io/docs#699]
We often get asked "what are the minimum requirements for running [managed] resin on my device?" We should add a response to this question to our FAQs. 

I've heard that we "recommend" at least 512 MB RAM and 4 GB storage but we should confirm and may want to present some ranges. There may also be other minimum stats we should recommend beyond RAM and storage.

**Comment by lurch** [comment]
It's a bit of a glib answer, but given that resinos runs on the Raspberry Pi Zero and Raspberry Pi Zero W, which only cost $5 and $10 respectively, it's hard to get more "minimum" than that :wink: 

Yes, I realise this is _actually_ asking about technical requirements. I guess it links in with https://docs.resin.io/hardware/devices/

**Comment by alisondavis17** [comment]
Yeah, that's usually the short answer we give, and that could even be included in this FAQ answer :) and it's also about theoretical support, not just the devices we already support today - it's a good initial filter for "can resin support my board?"

**Comment by lurch** [comment]
I guess "ability to run Linux and ~~Docker~~ Balena" will be one of the primary factors?

**Comment by alisondavis17** [comment]
Yes, that's usually the first thing we say, but people usually want to know sizes as well.
